core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -73,6 +73,8 @@ public class PartitionsTable extends BaseMetadataTable {
                 "equality_delete_file_count",
                 Types.IntegerType.get(),
                 "Count of equality delete files"),
+            Types.NestedField.required(
+                11, "total_data_file_size_in_bytes", Types.LongType.get(), "Total size in bytes"),
```

Comment:
This is still not changed back?  "Total size in bytes of data files"  Sorry if its still pending

Reply:
and also let's move it up to between 3 and 5 since it belong to data file group